### ISSUE TITLE
Accessibility - Lang attribute missing from html causing english text to be read with Norwegian (default for OS) voicing

### ISSUE THREAD

**Issue by maur1** [DependencyTrack/frontend#1204]
### Current Behavior

Lang attribute missing from html causing english text to be read with Norwegian (default for OS) voicing

### Proposed Behavior

Add the lang attribute to the HTML document, so that english text is read with English voicing in screen readers. Consider dynamic attributes if the frontend supports multiple languages

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [x] I have checked the [existing issues](https://github.com/DependencyTrack/frontend/issues) for whether this enhancement was already requested
